action.yaml

```diff
@@ -44,6 +44,7 @@ runs:
         access: public
 
     - name: Create Release
+      if: ${{ steps.publish.outputs.type }}
```

Comment:
just checking if the output is present? or do you need to check if it is something specific

Reply:
Just if it is present.  If a change in the version occurs this will be populated.  The docs state:
```
[Semver release type](https://github.com/npm/node-semver#release_types), initial if first release, different if other change, or empty if no release.
```

Reply:
ah it's `undefined` if no release is created
<img width="647" alt="image" src="https://github.com/PrefectHQ/actions-release-ui-components/assets/19556538/751f807b-4fad-4d9a-addc-6bf08be8b0ce">